node/lib/DeviceAccessor.ts

```diff
@@ -341,7 +347,15 @@ export class DeviceAccessor {
 
     private static capitalize(propertyName: string) {
         if (propertyName.length > 1) {
-            return propertyName.substr(0, 1).toUpperCase() + propertyName.substr(1);
+            return propertyName[0].toUpperCase() + propertyName.substr(1);
+        }
+
+        return propertyName;
+    }
+
+    private static uncapitalize(propertyName: string) {
+        if (propertyName.length > 1) {
+            return propertyName[0].toLowerCase() + propertyName.substr(1);
         }
```

Comment:
hmm.. yeah. it is just first character. 

---
In reply to: [73988756](https://github.com/openT2T/opent2t/pull/15#discussion_r73988756) [](ancestors = 73988756,73988719,73968450)